ubuntudesign/developer.ubuntu.com
# Use the get-feeds module to provide fallbacks on error

QA
--

Do `./run`, open the site and check feeds load.

Swap out `insights.ubuntu.com` URLs for `example.com`.

Now do `./run`, open the site, and check feeds show an error message, but that it doesn't break the whole page.
